lib/maths/CGammaRateConjugate.cc

```diff
@@ -1439,6 +1450,21 @@ void CGammaRateConjugate::print(const std::string& indent, std::string& result)
         return;
     }
 
+    std::string meanStr{"<unknown>"};
+    std::string sdStr{"<unknown>"};
```

Comment:
It might be worth considering if `<unknown>` is the best string to use to represent unknown values.  Is the likely client for reading the values Python?  If so, would some other string be easier for Python to interpret as meaning unknown?

Same in the other places where `<unknown>` is used.

The other thing about this is that using the string literal `<unknown>` in all the places means that:

1. If we ever want to change it we've got to grep for it and
2. There's a call to `strlen` on every initialisation

You could solve these problems by declaring a `const std::string` containing the unknown value.  That will also store the length making copying more efficient and can easily be changed if we decide the primary client is, say, JavaScript rather than Python.